classes/PodsInit.php

```diff
@@ -485,13 +485,17 @@ public function register_assets() {
 		}
 
 		wp_register_script(
-			'pods', PODS_URL . 'ui/js/jquery.pods.js', array(
+			'pods',
+			PODS_URL . 'ui/js/jquery.pods.js',
+			array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 0f7dcda6994885a99052161a796a4645</sub>